src/components/CountryCard.tsx

```diff
@@ -0,0 +1,50 @@
+import styles from './CountryCard.module.sass';
+import Image from 'next/image';
+
+export default function CountryCard({
+  flags,
+  name,
+  population,
+  region,
+  capital,
+}: {
+  flags: {
+    svg: string;
+    alt: string;
+  };
+  name: string;
+  population: number;
+  region: string;
+  capital: string;
+}) {
+  return (
+    <section className={styles.container}>
+      <div className={styles.imageContainer}>
+        <Image
```

Comment:
![image](https://github.com/Muneyy/rest-countries-api-next/assets/61583701/9bb27b18-5a6a-4ec5-a37e-4672a5079b5a)
I think the cards should be the same size what do you think?